pupil-spa/gen_config.sh

```diff
@@ -1,8 +1,13 @@
 #!/bin/bash
-apiUrl=${API_URL:-"http://localhost:3001"}
-authUrl=${AUTH_URL:-"http://localhost:3001/api/questions"}
-checkStartedUrl=${CHECK_STARTED_URL:-"http://localhost:3001/api/check-started"}
-checkSubmissionUrl=${CHECK_SUBMISSION_URL:-"http://localhost:3001/api/completed-check"}
+featureUseHpa=${FEATURE_USE_HPA:-"false"}
+if [[ "$featureUseHpa" == "true" ]]; then
```

Comment:
@jon-shipley if `featureUseHpa` is true then aside from `authUrl` remain undefined.
Is that intentional?
I would assume this would break the app at this stage right on check start.